.github/workflows/build-master.yaml.orig

```diff
@@ -0,0 +1,1112 @@
+
```

Comment:
Looks like a leftover?

Reply:
Yes I had some garbage in my checkout and I did a wild `git add`